docs/characterization/parallel_xeb.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>If you name the variable circuits_library, this notebook will be much more clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>changed</p>